src/_tests/__snapshots__/render.test.tsx-handles-some-simple-jsx.svg

```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <svg width="1024" height="768" xmlns="http://www.w3.org/2000/svg" version="1.1">
 
- <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="40" y="40" width="100" height="200"/>
+ <rect type="View" x="40" y="40" width="100" height="200" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
 
  <g transform='translate(40, 40)'>
-   <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="20" y="0" width="20" height="20"/>
-   <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="20" y="40" width="160" height="40"/>
+   <rect type="View" x="20" y="0" width="20" height="20" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
+   <rect type="View" x="20" y="40" width="160" height="40" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
 
    <g transform='translate(20, 40)'>
-     <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="10" y="20" width="10" height="10"/>
-     <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="20" y="40" width="10" height="10"/>
+     <rect type="View" x="10" y="20" width="10" height="10" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
+     <rect type="View" x="20" y="40" width="10" height="10" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
    </g>
 
-   <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="20" y="80" width="20" height="20"/>
-   <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="20" y="120" width="160" height="40"/>
+   <rect type="View" x="20" y="80" width="20" height="20" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
+   <rect type="View" x="20" y="120" width="160" height="40" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
 
    <g transform='translate(20, 120)'>
-     <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="20" y="0" width="10" height="10"/>
-     <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="10" y="20" width="10" height="10"/>
-     <rect type="View" fill-opacity="0.1" stroke-width="1" stroke="black" x="0" y="40" width="10" height="10"/>
+     <rect type="View" x="20" y="0" width="10" height="10" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
+     <rect type="View" x="10" y="20" width="10" height="10" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
+     <rect type="View" x="0" y="40" width="10" height="10" fill="none" stroke="black" stroke-width="0" rx="0" ry="0"/>
    </g>
 
  </g>
```

Comment:
Something that'll need to be addressed is what to show when there is a lack of  any border settings, almost all of these SVGs are not useful for visual regressions _unless_ you have borders set on each element.

<img width="1001" alt="screen shot 2017-10-16 at 4 19 19 pm" src="https://user-images.githubusercontent.com/49038/31633309-efa48ddc-b28d-11e7-8f05-a0d07423a22f.png">

Reply:
I’m not sure I follow. In RN, you wouldn’t see anything either.

Reply:
Is the idea to demonstrate layout?

Reply:
Yeah, I built this project specifically for visualizing layouts ( see the [VISION file](https://github.com/jest-community/jest-snapshots-svg/blob/master/VISION.md) ) removing that ability basically makes it unusable for what I started it for

Reply:
Got you! How would you feel about having a wireframe layout mode that shows all boxes, and a visual layout mode that tries to look at close to the actual design as we can?

Reply:
Yep, fine by me 👍